Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -68,6 +107,27 @@ extension BrowserViewController {
             notifyHttpsUpgrade(theme: Theme.of(selectedTab))
             notificationShown = true
         }
+        
+        // Step 5: Share Brave Benchmark Tiers
+        guard !notificationShown else { return }
+
+        if !Preferences.ProductNotificationBenchmarks.allTiersShown.value {
```

Comment:
what if you could move this condition up to have `allTiersShown` and `notificationShown` under one early return?
something like `if notificationShown || allTiersShown { return }`

this would remove one level of indentation

Reply:
Makes sense, usage is updated as 
`        if Preferences.ProductNotificationBenchmarks.allTiersShown.value && notificationShown { return }
`